packages/debounce-handler/src/index.js

```diff
@@ -7,9 +7,11 @@ const debounceHandler = (handlerName, delay, leadingCall) => (Target) => {
     constructor (props, context) {
       super(props, context)
 
+      const delayValue = typeof delay === 'function' ? delay(props) : delay
```

Comment:
We could do this in throttle as well.

Reply:
Agreed. It also lacks the same changes as in https://github.com/deepsweet/hocs/pull/22 :) Do you mind to work on that?

Reply:
Let's keep it "duplicated" for a while. If at some point we have a 3rd one then yes, I agree :)

Reply:
10-4 , I am down with that rule of 3 is a good one.

Reply:
Whats the command line paradigm to update jest snapshots with `start`? I tried `yarn start test throttle-handler --updateSnapshot` and `yarn start test throttle-handler -u` with no success .

Reply:
Hm, I never used it like this, I should probably propagate an argument.

`yarn start testWatch` + `u` keypress when it comes to it.